included/9.7.0/Dockerfile

```diff
@@ -0,0 +1,58 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 9.7.0 cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+#
+# build this image with command
+#   docker build -t cypress/included:9.7.0 .
+#
+FROM cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
```

Comment:
Is slim the correct image? @admah did you say this dropped the git dependency?

Reply:
@emilyrohrbough this is the correct image. You can see that `git` is included [here](https://github.com/cypress-io/cypress-docker-images/blob/40c5e6a0516dd2e1fdf57fa3b1271a54f764404d/browsers/node16.14.2-slim-chrome100-ff99-edge/Dockerfile#L19).

Reply:
Cool cool!